src/lib/ApiClient.js

```diff
@@ -991,6 +1008,34 @@ class ApiClient {
 		}
 		return false;
 	}
+
+	normalizedApiError(response) {
+		if (_.isError(response) || response instanceof VError) {
+			return response;
+		}
+
+		let reason = 'Server error';
+		if (typeof response === 'string') {
+			reason = response;
+		} else if (response.errors) {
+			reason = response.errors.map((err) => {
+				if (err.error) {
+					if (err.error.status) {
+						return err.error.status;
+					} else {
+						return err.error;
+					}
+				} else {
+					return err;
+				}
```

Comment:
Lines 1022 -> 1030.

We can replace the nested if statements with
```
return err.error && err.error.status || err.error || err; 
```
![image](https://user-images.githubusercontent.com/14219325/41112323-49d9a218-6a87-11e8-8406-53fd8d35f5ea.png)

Reply:
I'd like to see some parentheses since the priority of operators tends to play tricks with these long boolean chains.

Reply:
`return (err.error && err.error.status) || err.error || err; `